### ISSUE TITLE
List community member biographies and socials

### ISSUE THREAD

**Issue by BOLT04** [EddieJaoudeCommunity/EddieBotUI#26]
**Is your feature request related to a problem? Please describe.**
We are not making use of the `^bio` command of EddieBot. This feature lets users have their Github, LinkedIn, and other socials stored in the DB, but they are not shown anywhere unless someone inputs the command to read their bio. One thing to be concerned about perhaps is that members of the community need to be aware this information will be public. Or at least used in this project, so we could add a link to the production app of EddieBotUI so they know where it's being used 👍 .

**Describe the solution you'd like**
We can have a new navigation item for "Community" or "Members". When the user navigates to it, we have a grid of members and their bios. There would be a card UI, that expands to give a more detailed description of their bio (since some users have a long bio on discord).

**Describe alternatives you've considered**
N/A

**Additional context**
![imagem](https://user-images.githubusercontent.com/18630253/93687134-46139580-fab3-11ea-8753-c6529139304d.png)

**Comment by eddiejaoude** [comment]
Great idea 👍 - I think we should switch DB first, otherwise there will be more updates to do afterwards and we already have a few places to update.

**Comment by BOLT04** [comment]
@eddiejaoude completely agreed 👍. I created the issue to have it on the backlog and the community can see that we plan on doing this. I still think it's possible to build a UI that does not depend on any code from the DB (separation of concerns 💯) with mock data, but the DB switch has a higher priority than this task.

Still, anyone is welcome to contribute when they can 😃.

**Comment by eddiejaoude** [comment]
Yeah I think we need an API to manage all the data access, that way we can mock it out like you said 👍
